deployment/https/env.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+export DB_PW_PROJECTBUILDER=tobedefined
```

Comment:
Why dont you leave that empty? Then the executor should call a warning, right?

Reply:
done